### ISSUE TITLE
Drone run fails "Internal Server Error"

### ISSUE THREAD

**Issue by JeanMertz** [drone/drone#775]
Whenever I build using a custom image, I get the following error:

```
Status: Downloaded newer image for jeanmertz/droneio-ci-stack:latest
starting build 
2014/12/30 18:27:55 Error building 6ea06bcba4ce461c0849ce50f60a57b485ef4d6c, Err: Internal Server Error
2014/12/30 18:27:56 subscription's close channel received message
```

This is after the image pull was completed successfully.

Is there any way to get some more details on what exactly caused this error? (it happens consistently whenever I retry the build)

**Comment by bradrydzewski** [comment]
@JeanMertz were you able to test after I merged #777 and get a detailed error message? Is the issue persisting?

**Comment by bradrydzewski** [comment]
If the issue persists and we can get the logs we can re-open

**Comment by juliocesar-io** [comment]
I tested after the merged #777 and the issue persists.

```
91c95931e552: Pull complete
...
...
..
Status: Downloaded newer image for hello-world:latest
starting build 
2015/06/27 20:44:19 Error building 08a934bbe8b54a099e67bdccad576a6c88f16fd4, Err: Error: Failed to start build container. Internal Server Error
2015/06/27 20:44:20 subscription's close channel received message
```

I'm using CoreOs stable (681.2.0) with Docker client and server 1.6.2 and Drone 0.3.0-alpha. 

Notice that the image pulls correctly as @JeanMertz mention before. Any thoughts ?
